examples/04-lhotse-shar.ipynb

```diff
@@ -0,0 +1,882 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    shards = cuts_train.to_shar(data_dir, fields={"recording": "wav"}, shard_size=1000)
</pre><p>May be helpful to explain what other values may be passed to the <code>fields</code> dict.</p><p>Also, what is the granularity of sharding? Is each recording treated as a unit?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lhotse-speech/lhotse/pull/1006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yes, each cut is treated as a unit, so shard_size=1000 means 1000 cuts per shard. I will add this explanation to the tutorial and also provide link to the documentation that describes the `fields` dict in more detail (I noticed that Lhotse Shar docs were not linked in readthedocs and I'm also fixing it now).